README.md

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
Please follow the same logic of the comments of your other PR and adjust this PR as well